src/core/range.c

```diff
@@ -278,7 +280,7 @@ QuicRangeAddRange(
                     QuicRangeCompare(&Key, Sub) == 0) {
                 --i;
             }
-            Sub = QuicRangeGet(Range, i);
+            QuicRangeGet(Range, i);
```

Comment:
This doesn't look right, does it?

Reply:
Well, I guess it's never being read, so the whole line should be removed?

Reply:
Its unconditionally overwritten on line 294, so I understand why its flagged. If that is the case, is this call then needed?

Reply:
I wasn't sure if the call has side effects. I guess not.